examples/migration/create_complete_campaign_both_apis_phase_2.py

```diff
@@ -27,16 +27,13 @@
 """
 
 import argparse
-import collections
 import datetime
-import sys
 import urllib.parse
 import uuid
 
 from googleads import adwords
 
 from google.ads.google_ads.client import GoogleAdsClient
-from google.ads.google_ads.errors import GoogleAdsException
```

Comment:
instead of removing the exception, a try/catch block should be added to `__main__`

Reply:
Added.